Dockerfile

```diff
@@ -180,6 +180,13 @@ CMD ["server"]
 # Stage 2 - Debian
 FROM debian:${DEBIAN_TAG} AS debian
 
+# Add atlantis user to Debian as well
+RUN useradd --create-home --user-group --shell /bin/bash atlantis && \
```

Comment:
happy to make this have a stable uid / gid if folks think it's useful here.

Reply:
I think in the alpine this was 1001 I believe

Reply:
I could not find any reference on that so I think is ok

Reply:
fwiw, I believe (older) alpine images were uid=100, gid=1000
```
$ cat /etc/passwd  | grep atlantis
atlantis:x:100:1000
```